 * Serving Flask app "api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
Traceback (most recent call last):
  File "app.py", line 4, in <module>
    app.run(host='0.0.0.0', port=80)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 990, in run
    run_simple(host, port, self, **options)
  File "/usr/local/lib/python3.6/site-packages/werkzeug/serving.py", line 1030, in run_simple
    s.bind(server_address)
PermissionError: [Errno 13] Permission denied
 * Serving Flask app "api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [04/Aug/2020 18:43:03] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:29] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:30] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:30] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:30] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:31] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:42] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:43] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:45] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:47] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:49] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:51] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:43:52] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:00] "[37mPOST /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:07] "[31m[1mPOST /signup/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [04/Aug/2020 18:44:08] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:12] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:17] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:21] "[37mGET /user/example HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:24] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:28] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:36] "[37mPOST /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:39] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:40] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:42] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:44] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:47] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:53] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:56] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:57] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:44:58] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:00] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:01] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:02] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:02] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:06] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:08] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:09] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:10] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:13] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:17] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:22] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:24] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:29] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:31] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:32] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:43] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:51] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:54] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:45:56] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:06] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:09] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:12] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:21] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:23] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:36] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:43] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:46:46] "[37mGET / HTTP/1.1[0m" 200 -
 * Serving Flask app "api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [04/Aug/2020 18:48:17] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:02] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:05] "[37mGET /user/user3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:07] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:08] "[37mGET /user/user3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:10] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:12] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:28] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:30] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:49:57] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:15] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:16] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:19] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:23] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:29] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:31] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:35] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:36] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:50:59] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:01] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:04] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:13] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:15] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:17] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:19] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [04/Aug/2020 18:51:31] "[37mGET / HTTP/1.1[0m" 200 -
90.76.35.18 - - [04/Aug/2020 19:15:27] "[37mGET / HTTP/1.1[0m" 200 -
2.236.136.12 - - [04/Aug/2020 19:25:17] "[37mGET / HTTP/1.1[0m" 200 -
109.111.139.6 - - [04/Aug/2020 19:42:07] "[37mGET / HTTP/1.1[0m" 200 -
182.138.137.176 - - [04/Aug/2020 20:09:55] "[37mGET / HTTP/1.1[0m" 200 -
223.166.74.123 - - [04/Aug/2020 20:09:57] "[37mGET / HTTP/1.1[0m" 200 -
182.138.137.156 - - [04/Aug/2020 20:09:57] "[33mGET /currentsetting.htm HTTP/1.1[0m" 404 -
123.191.132.172 - - [04/Aug/2020 20:09:57] "[33mGET /scgi-bin/platform.cgi HTTP/1.1[0m" 404 -
182.138.137.26 - - [04/Aug/2020 20:09:58] "[33mGET /cgi-bin/p2p.cgi?cmd=p2p.cgi&-action=get HTTP/1.1[0m" 404 -
221.13.12.33 - - [04/Aug/2020 20:09:59] "[37mGET / HTTP/1.1[0m" 200 -
49.74.97.151 - - [04/Aug/2020 20:09:59] "[33mGET /login.html HTTP/1.1[0m" 404 -
175.184.164.140 - - [04/Aug/2020 20:09:59] "[37mGET / HTTP/1.1[0m" 200 -
182.101.49.67 - - [04/Aug/2020 20:10:00] "[33mGET /cgi-bin/welcome.cgi HTTP/1.1[0m" 404 -
110.177.177.179 - - [04/Aug/2020 20:10:00] "[33mGET /index.html HTTP/1.1[0m" 404 -
60.13.136.3 - - [04/Aug/2020 20:10:00] "[37mGET / HTTP/1.1[0m" 200 -
124.88.55.29 - - [04/Aug/2020 20:10:01] "[33mGET /cgi-bin/user/Config.cgi?.cab&action=get&category=Account.* HTTP/1.1[0m" 404 -
106.45.1.153 - - [04/Aug/2020 20:10:01] "[33mGET /cgi-bin/nobody/Machine.cgi?action=get_capability HTTP/1.1[0m" 404 -
221.13.12.115 - - [04/Aug/2020 20:10:01] "[33mGET /current_config/passwd HTTP/1.1[0m" 404 -
36.47.162.45 - - [04/Aug/2020 20:10:02] "[33mGET /system.ini?loginuse&loginpas HTTP/1.1[0m" 404 -
222.94.140.215 - - [04/Aug/2020 20:10:02] "[37mGET / HTTP/1.1[0m" 200 -
110.177.183.156 - - [04/Aug/2020 20:10:02] "[33mGET /cgi-bin/webproc HTTP/1.1[0m" 404 -
223.166.74.44 - - [04/Aug/2020 20:10:02] "[33mGET /webaccess/index.php HTTP/1.1[0m" 404 -
222.79.50.35 - - [04/Aug/2020 20:10:04] "[33mGET /HNAP1/ HTTP/1.1[0m" 404 -
101.249.53.202 - - [04/Aug/2020 20:10:04] "[33mGET /common/info.cgi HTTP/1.1[0m" 404 -
49.74.98.58 - - [04/Aug/2020 20:10:05] "[33mGET /hnap/hnap_service HTTP/1.1[0m" 404 -
112.66.98.244 - - [04/Aug/2020 20:10:06] "[33mGET /stssys.htm HTTP/1.1[0m" 404 -
58.19.92.11 - - [04/Aug/2020 20:10:06] "[33mGET /index.asp HTTP/1.1[0m" 404 -
123.245.25.93 - - [04/Aug/2020 20:10:07] "[33mGET /login.htm HTTP/1.1[0m" 404 -
119.118.20.205 - - [04/Aug/2020 20:10:07] "[37mGET / HTTP/1.1[0m" 200 -
123.145.29.24 - - [04/Aug/2020 20:10:07] "[33mGET /webpages/login.html HTTP/1.1[0m" 404 -
113.206.182.41 - - [04/Aug/2020 20:10:07] "[37mGET / HTTP/1.1[0m" 200 -
49.118.203.50 - - [04/Aug/2020 20:10:08] "[37mGET / HTTP/1.1[0m" 200 -
116.113.36.128 - - [04/Aug/2020 20:10:08] "[33mGET /FHFactoryCheck.html HTTP/1.1[0m" 404 -
101.24.131.18 - - [04/Aug/2020 20:10:08] "[37mGET / HTTP/1.1[0m" 200 -
125.84.180.84 - - [04/Aug/2020 20:10:09] "[37mGET / HTTP/1.1[0m" 200 -
27.224.137.69 - - [04/Aug/2020 20:10:10] "[37mGET / HTTP/1.1[0m" 200 -
188.216.82.24 - - [04/Aug/2020 21:46:58] "[37mGET / HTTP/1.1[0m" 200 -
128.14.134.134 - - [04/Aug/2020 22:08:03] "[37mGET / HTTP/1.1[0m" 200 -
64.227.97.195 - - [04/Aug/2020 23:10:43] code 400, message Bad request syntax ('\x00\x0e8oÌì\x10±Rû\x89\x00\x00\x00\x00\x00')
64.227.97.195 - - [04/Aug/2020 23:10:43] "[35m[1m 8oÌì±Rû     [0m" HTTPStatus.BAD_REQUEST -
178.73.215.171 - - [04/Aug/2020 23:50:48] "[37mGET / HTTP/1.0[0m" 200 -
162.243.129.17 - - [05/Aug/2020 00:37:37] "[33mGET /UPnP/IGD.xml HTTP/1.1[0m" 404 -
91.80.136.81 - - [05/Aug/2020 01:38:04] "[33mGET /phpmyadmin/ HTTP/1.1[0m" 404 -
185.39.11.105 - - [05/Aug/2020 02:05:55] "[33mGET /config/getuser?index=0 HTTP/1.1[0m" 404 -
151.235.217.235 - - [05/Aug/2020 02:20:47] "[37mGET / HTTP/1.1[0m" 200 -
151.235.217.235 - - [05/Aug/2020 02:20:47] "[37mGET / HTTP/1.1[0m" 200 -
218.161.31.146 - - [05/Aug/2020 02:39:37] "[37mGET / HTTP/1.1[0m" 200 -
185.216.140.6 - - [05/Aug/2020 03:12:20] "[37mGET / HTTP/1.1[0m" 200 -
35.183.203.62 - - [05/Aug/2020 03:40:38] "[33mGET /.git/HEAD HTTP/1.1[0m" 404 -
193.118.55.146 - - [05/Aug/2020 03:52:10] "[37mGET / HTTP/1.1[0m" 200 -
72.252.117.87 - - [05/Aug/2020 04:06:13] "[33mGET /phpmyadmin/ HTTP/1.1[0m" 404 -
113.31.115.156 - - [05/Aug/2020 04:30:30] "[37mGET / HTTP/1.1[0m" 200 -
113.31.115.156 - - [05/Aug/2020 04:30:30] "[37mGET / HTTP/1.1[0m" 200 -
88.249.194.237 - - [05/Aug/2020 04:32:54] "[37mGET / HTTP/1.0[0m" 200 -
114.33.90.102 - - [05/Aug/2020 04:52:15] "[37mGET / HTTP/1.1[0m" 200 -
172.104.108.109 - - [05/Aug/2020 06:24:54] "[37mGET / HTTP/1.1[0m" 200 -
220.133.57.67 - - [05/Aug/2020 08:41:00] "[37mGET / HTTP/1.1[0m" 200 -
195.54.160.21 - - [05/Aug/2020 09:25:24] "[33mGET /solr/admin/info/system?wt=json HTTP/1.1[0m" 404 -
195.54.160.21 - - [05/Aug/2020 09:42:25] "[37mGET /?XDEBUG_SESSION_START=phpstorm HTTP/1.1[0m" 200 -
195.54.160.21 - - [05/Aug/2020 10:08:57] "[37mGET /?a=fetch&content=<php>die(@md5(HelloThinkCMF))</php> HTTP/1.1[0m" 200 -
128.14.134.134 - - [05/Aug/2020 10:11:50] "[37mGET / HTTP/1.1[0m" 200 -
195.54.160.21 - - [05/Aug/2020 10:15:49] "[33mGET /index.php?s=/Index/\think\app/invokefunction&function=call_user_func_array&vars[0]=md5&vars[1][]=HelloThinkPHP HTTP/1.1[0m" 404 -
185.39.11.105 - - [05/Aug/2020 11:02:19] "[37mGET http://example.com/ HTTP/1.1[0m" 200 -
176.126.175.10 - - [05/Aug/2020 11:44:48] "[37mGET / HTTP/1.1[0m" 200 -
87.250.108.62 - - [05/Aug/2020 12:33:26] "[37mGET / HTTP/1.1[0m" 200 -
23.90.145.40 - - [05/Aug/2020 12:51:42] "[37mGET / HTTP/1.0[0m" 200 -
23.90.145.40 - - [05/Aug/2020 12:51:42] code 400, message Bad request version ('À\x14À')
23.90.145.40 - - [05/Aug/2020 12:51:42] "[35m[1m   {ÜÄÉz6òÝ'ÂUîáª<¾wX }ùI{Yð  À/À+À0À,ÀÀ	ÀÀ[0m" HTTPStatus.BAD_REQUEST -
196.52.43.109 - - [05/Aug/2020 13:14:24] "[37mGET / HTTP/1.1[0m" 200 -
122.228.19.79 - - [05/Aug/2020 13:24:28] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 14:29:38] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 14:29:39] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 14:29:40] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 14:31:38] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 14:31:40] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
195.54.160.21 - - [05/Aug/2020 14:59:17] "[33mPOST /api/jsonws/invoke HTTP/1.1[0m" 404 -
164.70.183.224 - - [05/Aug/2020 15:10:46] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:10:53] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:10:56] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:11:13] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:11:51] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:12:25] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:12:43] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:12:59] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:15:49] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:15:52] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:16:15] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:16:29] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:20:11] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:21:07] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:21:19] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:22:20] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:23:11] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:23:26] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:23:51] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:24:06] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:24:34] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:25:19] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:25:56] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:26:39] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:26:42] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:26:57] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:27:54] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:27:56] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:28:22] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:28:37] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:29:04] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:29:58] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:30:51] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:30:59] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:31:05] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:31:09] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:31:19] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:32:05] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:32:38] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:11] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:12] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:30] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:31] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:42] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:48] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:50] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:35:56] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:04] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:08] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:10] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:15] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:22] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:23] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:41] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:42] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:46] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:48] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:36:51] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:37:38] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:37:49] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:37:51] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:37:55] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:37:57] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:02] "[37mGET /user/user3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:05] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:09] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:13] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:26] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:28] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:46] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:50] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:39:55] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:45:10] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:45:14] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:49:44] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:49:46] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:49:50] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:49:54] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:01] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:02] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:04] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:06] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:08] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:13] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:21] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:30] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:34] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:44] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:45] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:46] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:52] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:55] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:58] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:50:59] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:01] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:22] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:23] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:33] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:37] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:39] "[37mGET /user/test3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:41] "[37mGET /user/test4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:42] "[37mGET /user/test5 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:44] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:47] "[37mGET /user/example HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:49] "[37mGET /user/user3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:53] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:54] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:51:58] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:00] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:04] "[37mPOST /report/delete/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:10] "[37mPOST /report/delete/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:13] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:15] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:26] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:27] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:33] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:38] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:43] "[37mPOST /report/delete/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:46] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:47] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:51] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:55] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:57] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:52:59] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:53:03] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:53:05] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:53:26] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:53:37] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:53:54] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:53:57] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:54:00] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:54:04] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:54:08] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:56:08] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:56:25] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:57:56] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:57:58] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 15:59:35] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:00:24] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:00:31] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:00:33] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:00:45] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:00:58] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:01:07] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:01:30] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:08] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:09] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:10] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:12] "[35m[1mGET /user/matsuoka HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 140, in showUserPage
    'follower_id': current_user.id,
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 16:02:12] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:12] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:12] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:12] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:12] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:12] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:15] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:20] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:22] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:33] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:34] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:35] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:35] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:39] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:02:55] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:03:26] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:03:31] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:03:34] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:03:37] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:04:57] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:05:06] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:05:09] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:06:10] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:07:51] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:08:02] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:08:06] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:09:25] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:11:28] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:12:33] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:12:35] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:13:27] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:29:02] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:29:18] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:29:20] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:29:26] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
176.126.175.10 - - [05/Aug/2020 16:31:40] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:45:16] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:46:12] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:46:26] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:46:50] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:46:53] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:46:55] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:47:55] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:48:03] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:10] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:20] "[35m[1mGET /user/matsuoka HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 140, in showUserPage
    'follower_id': current_user.id,
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 16:51:20] "[37mGET /user/matsuoka?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:21] "[35m[1mGET /reports/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 223, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 16:51:21] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:21] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:21] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:22] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:22] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:22] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:23] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:26] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:28] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:51:35] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:52:07] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:52:32] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:52:40] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:53:19] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:53:49] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
71.6.232.4 - - [05/Aug/2020 16:54:00] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:54:04] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:54:34] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:54:58] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:55:33] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:55:52] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:57:24] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:57:27] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:57:28] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:57:44] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:57:54] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:04] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:05] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:07] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:15] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:18] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:20] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:21] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:30] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:35] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:36] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:58:39] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:11] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:14] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:16] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:24] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:25] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:33] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:37] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:39] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 16:59:46] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:00:03] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:00:36] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:00:44] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:00:55] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:00:56] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:01:00] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:01:19] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:01:21] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:01:23] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:01:25] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:07] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:09] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:11] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:12] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:15] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:16] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:18] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:02:25] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:08] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:10] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:11] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:13] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:18] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:20] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:23] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:26] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:43] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:45] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:47] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:49] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:51] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:54] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:05:58] "[37mGET /user/matsuoka/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:06:01] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:07:07] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:07:09] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:07:20] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:07:22] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:08:29] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:08:30] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:08:31] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:08:32] "[37mGET /reports/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
2
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:08:59] "[33mGET /reports/ HTTP/1.1[0m" 404 -
164.70.183.224 - - [05/Aug/2020 17:09:02] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:09:07] "[33mGET /reports/ HTTP/1.1[0m" 404 -
164.70.183.224 - - [05/Aug/2020 17:09:35] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:09:37] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:09:39] "[37mGET / HTTP/1.1[0m" 200 -
37.49.230.35 - - [05/Aug/2020 17:09:55] code 400, message Bad request syntax ('{D79E94C5-70F0-46BD-965B-E17497CCB598}')
37.49.230.35 - - [05/Aug/2020 17:09:55] "[35m[1m{D79E94C5-70F0-46BD-965B-E17497CCB598}[0m" HTTPStatus.BAD_REQUEST -
164.70.183.224 - - [05/Aug/2020 17:10:33] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:10:34] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:10:39] "[37mPOST /login/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:11:51] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:11:55] "[37mPOST /login/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:13:47] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 67, in login
    showReports()
  File "/home/dbuser/dblec_app/api/views/user.py", line 224, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:13:47] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:13:47] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:13:47] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:13:47] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:13:47] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:13:47] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:50] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:53] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 67, in login
    showReports()
  File "/home/dbuser/dblec_app/api/views/user.py", line 224, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:14:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:14:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:15:59] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 68, in login
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:15:59] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:15:59] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:15:59] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:15:59] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:15:59] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:15:59] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:02] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 68, in login
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:16:02] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:02] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:02] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:02] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:02] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:02] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:16:17] "[37mPOST /login/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:16:30] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:45] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:50] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:52] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:56] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:16:58] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:18:54] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 68, in login
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:18:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:18:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:18:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:18:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:18:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:18:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:20:42] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 68, in login
    user = getFilteredUser(user['name'])
NameError: name 'getFilteredUser' is not defined
164.70.183.224 - - [05/Aug/2020 17:20:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:20:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:20:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:20:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:20:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:20:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:21:12] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 70, in login
    user_id = user_schema.dump(user).id
AttributeError: 'list' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:21:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:21:35] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 70, in login
    user_id = user.id
AttributeError: 'BaseQuery' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:21:35] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:35] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:35] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:35] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:35] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:21:35] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:22:10] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 70, in login
    user_id = user_schema.dump(user)[0].id
AttributeError: 'dict' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:22:10] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:10] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:10] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:11] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:11] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:11] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:22:20] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 70, in login
    user_id = user_schema.dump(user)[0][id]
KeyError: <built-in function id>
164.70.183.224 - - [05/Aug/2020 17:22:20] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:20] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:20] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:21] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:21] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:22:21] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:23:12] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 71, in login
    follower_list = Follow.getFollowerList(user_id.id)
AttributeError: 'dict' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:23:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:23:22] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:24] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:25] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:23:30] "[37mPOST /login/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:25:53] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:25:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:25:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:25:53] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:25:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:25:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:25:54] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:26:12] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:26:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:26:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:26:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:26:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:26:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:26:12] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:31] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:27:31] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:31] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:31] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:31] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:31] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:32] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:41] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:42] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:43] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:43] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:27:43] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:08] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:28:08] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:08] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:08] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:08] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:08] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:08] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:09] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:13] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:28:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:13] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:26] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:28:26] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:26] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:26] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:26] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:26] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:28:26] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Serving Flask app "api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:29:22] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:29:22] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:22] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:22] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:23] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:23] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:23] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:29:48] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:50] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:29:52] "[37mGET /logout/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:29:59] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:03] "[35m[1mPOST /login/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 81, in login
    return render_template("reports.html", reports=report_schema.dump(report), username=user['name'])
  File "/usr/local/lib64/python3.6/site-packages/sqlalchemy/orm/query.py", line 3176, in __getitem__
    return list(self[item : item + 1])[0]
TypeError: must be str, not int
164.70.183.224 - - [05/Aug/2020 17:30:03] "[37mGET /login/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:03] "[37mGET /login/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:03] "[37mGET /login/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:03] "[37mGET /login/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:03] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:03] "[37mGET /login/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:30:57] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:30:59] "[37mPOST /login/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:31:16] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:17] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:18] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:18] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:19] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:19] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:19] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:20] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:20] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:20] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:20] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:23] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:24] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:31:28] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:05] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:08] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:09] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:10] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:10] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:10] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:11] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:11] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:11] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:11] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:12] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:12] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:32:12] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:44] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:46] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:47] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:55] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:57] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:58] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:58] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:58] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:33:58] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:34:22] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:34:25] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:34:26] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:34:32] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:34:34] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:34:41] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:28] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:30] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:36] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:45] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:47] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:52] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:54] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:35:57] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:10] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:12] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:13] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:16] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:18] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:25] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:26] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:31] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:33] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:34] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:36] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:37] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:38] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:39] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:44] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:45] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:46] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:48] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:48] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:36:51] "[37mGET /signup/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:37:59] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:12] "[37mPOST /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:19] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:28] "[37mGET /user/testtest HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:31] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:32] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:33] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:34] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:38:39] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:39:01] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:39:30] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:39:43] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:39:47] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:39:51] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:18] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:20] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:26] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:37] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:42] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:56] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:57] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:40:59] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:01] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:02] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:08] "[31m[1mPOST /login/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [05/Aug/2020 17:41:12] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:28] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:38] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:39] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:40] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:44] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:46] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:47] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:48] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:54] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:41:55] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:01] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:02] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:03] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:08] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:34] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:38] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:41] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:46] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:42:52] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:08] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 243, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:44:08] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:09] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:09] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:09] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:09] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:19] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 243, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:44:19] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:19] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:19] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:19] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:19] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:29] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:34] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:36] "[31m[1mGET /logout/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [05/Aug/2020 17:44:42] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 243, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:44:42] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:42] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:42] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:42] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:42] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:42] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:53] "[31m[1mGET /logout/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [05/Aug/2020 17:44:56] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 243, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:44:56] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:56] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:56] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:56] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:56] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:57] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:59] "[35m[1mGET / HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 243, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:44:59] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:59] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:59] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:59] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:59] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:44:59] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
3
2
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:46:56] "[37mGET / HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:48:16] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:48:20] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:48:23] "[37mGET / HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:49:48] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:49:49] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:49:51] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:02] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:18] "[35m[1mGET /reports/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 244, in showReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 17:50:18] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:18] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:18] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:18] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:18] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:50:18] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:51:31] "[35m[1mGET /reports/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2098, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
164.70.183.224 - - [05/Aug/2020 17:51:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:51:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:51:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:51:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:51:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:51:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:52:30] "[35m[1mGET /reports/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2098, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
164.70.183.224 - - [05/Aug/2020 17:52:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:31] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:33] "[35m[1mGET /reports/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2098, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
164.70.183.224 - - [05/Aug/2020 17:52:33] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:33] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:33] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:33] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:33] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:33] "[37mGET /reports/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:52:44] "[37mGET /reports/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:52:46] "[37mGET /login/ HTTP/1.1[0m" 200 -
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
 * Detected change in '/home/dbuser/dblec_app/api/views/user.py', reloading
 * Restarting with stat
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Debugger is active!
 * Debugger PIN: 324-991-647
164.70.183.224 - - [05/Aug/2020 17:53:07] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:17] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:20] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:24] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:31] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:32] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:38] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:40] "[37mGET /user/test/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:44] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:53:46] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:54:49] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:54:51] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:54:55] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:54:59] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:01] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:04] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:05] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:07] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:09] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:10] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:14] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:15] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:17] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:19] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:20] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:21] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:24] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:47] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:48] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:51] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:56] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:55:59] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:03] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:06] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:08] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:22] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:24] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:26] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:56:37] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:00] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:01] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:02] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:05] "[37mGET /user/mizutani/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:16] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:48] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:54] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:57:57] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:06] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:28] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:31] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:44] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:51] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:56] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:58:57] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:01] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:17] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:21] "[37mGET /user/test/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:23] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:27] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:28] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 17:59:49] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:00:19] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:00:21] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:00:23] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:00:26] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:00:30] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:06:21] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:06:24] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:06:46] "[37mPOST /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:06:49] "[37mGET /user/test_user HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:06:56] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:00] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:02] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:05] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:07] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:08] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:11] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:13] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:18] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:20] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:21] "[37mGET /restaurant/4 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:23] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:07:25] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:11:30] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 18:12:36] "[37mGET / HTTP/1.1[0m" 200 -
106.72.138.160 - - [05/Aug/2020 18:40:38] "[37mGET / HTTP/1.1[0m" 200 -
106.72.138.160 - - [05/Aug/2020 18:40:42] "[37mGET /login/ HTTP/1.1[0m" 200 -
106.72.138.160 - - [05/Aug/2020 18:40:46] "[37mGET / HTTP/1.1[0m" 200 -
106.72.138.160 - - [05/Aug/2020 19:00:02] "[37mGET / HTTP/1.1[0m" 200 -
106.72.138.160 - - [05/Aug/2020 19:00:03] "[37mGET /login/ HTTP/1.1[0m" 200 -
106.72.138.160 - - [05/Aug/2020 19:00:13] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:34:15] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:34:16] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:34:19] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:34:28] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:34:29] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:34:31] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:35:54] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:35:56] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:35:58] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:36:05] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:36:35] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:36:48] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:37:18] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:37:25] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:38:28] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:38:29] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:38:31] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:38:39] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:38:44] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:47:31] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
86.104.10.49 - - [05/Aug/2020 19:49:36] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:51:31] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:55:44] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:03] "[37mGET /restaurant/5 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:26] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:29] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:31] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:34] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:36] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:43] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:45] "[37mGET /user/matsuoka/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:46] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:56:48] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:11] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:13] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:14] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:15] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:16] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:21] "[37mGET /restaurant/5 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:48] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:52] "[37mGET /restaurant/5 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 19:59:58] "[37mGET /restaurant/6 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:00:15] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:00:19] "[37mGET /restaurant/7 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:00:51] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:00:55] "[37mGET /restaurant/8 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:01:01] "[37mGET /restaurant/9 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:01:15] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:01:19] "[37mGET /restaurant/10 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:01:21] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:01:25] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:01:45] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:08] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:11] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:17] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:22] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:27] "[37mGET /user/test_user HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:28] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:32] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:33] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:36] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:38] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:02:52] "[37mGET / HTTP/1.1[0m" 200 -
162.243.128.132 - - [05/Aug/2020 20:03:30] "[33mGET /portal/redlion HTTP/1.1[0m" 404 -
164.70.183.224 - - [05/Aug/2020 20:04:18] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:20] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:27] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:30] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:36] "[37mGET /restaurant/5 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:39] "[37mGET /restaurant/6 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:41] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:47] "[37mGET /restaurant/8 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:49] "[37mGET /restaurant/9 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:52] "[37mGET /restaurant/6 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:04:55] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:01] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:06] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:27] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:29] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:32] "[37mGET /user/matsuoka/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:41] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:44] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:05:54] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:00] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:02] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:07] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:30] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:35] "[35m[1mGET /user/test HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 160, in showUserPage
    'follower_id': current_user.id,
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [05/Aug/2020 20:06:35] "[37mGET /user/test?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:35] "[37mGET /user/test?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:35] "[37mGET /user/test?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:35] "[37mGET /user/test?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:35] "[37mGET /user/test?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:35] "[37mGET /user/test?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:41] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:43] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:47] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:48] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:49] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:06:51] "[37mGET /logout/ HTTP/1.1[0m" 200 -
109.73.180.231 - - [05/Aug/2020 20:23:34] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:39:29] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:39:32] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:39:46] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:39:51] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:40:09] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 20:40:11] "[37mGET /logout/ HTTP/1.1[0m" 200 -
190.111.140.11 - - [05/Aug/2020 21:31:16] "[37mGET / HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:31:41] "[37mGET / HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:31:52] "[37mGET / HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:00] "[37mGET /logout/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:01] "[37mGET /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:04] "[37mPOST /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:19] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:20] "[37mGET /user/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:22] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:23] "[37mGET /logout/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:26] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:31] "[37mGET /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:32:34] "[37mPOST /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:35:01] "[37mGET /logout/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:35:02] "[37mGET /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:35:10] "[31m[1mPOST /login/ HTTP/1.1[0m" 401 -
221.115.2.35 - - [05/Aug/2020 22:35:14] "[31m[1mPOST /login/ HTTP/1.1[0m" 401 -
221.115.2.35 - - [05/Aug/2020 22:35:17] "[37mPOST /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:35:22] "[37mGET /user/test HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:35:24] "[37mGET / HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:36:40] "[37mGET /logout/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:39:42] "[37mGET /login/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:40:19] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:44:32] "[31m[1mGET /logout/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [05/Aug/2020 22:44:35] "[31m[1mGET /logout/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [05/Aug/2020 22:44:36] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:44:41] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:44:43] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:44:44] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:44:49] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:44:58] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:45:01] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:45:15] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:45:17] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:45:48] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:45:51] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:45:59] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:09] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:14] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:50] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:52] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:54] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:55] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 22:46:59] "[37mGET / HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:48:46] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:49:12] "[37mGET /user/mizutani/following HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:49:50] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:49:55] "[37mGET /user/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:49:56] "[37mGET /user/user2 HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:50:05] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:50:15] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:50:20] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:50:32] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:50:35] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:22] "[37mGET /user/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:23] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:26] "[37mGET /report/edit/?id=26 HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:36] "[37mGET /user/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:39] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:41] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
221.115.2.35 - - [05/Aug/2020 22:51:44] "[37mGET /report/edit/?id=26 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:25] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:27] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:35] "[37mPOST /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:39] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:40] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:42] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:44] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:46] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:48] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:50] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:52] "[37mGET /user/test HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:54] "[37mPOST /follow/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:15:56] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:00] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:02] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:04] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:10] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:13] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:21] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:27] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:35] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:40] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:44] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:16:55] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:17:04] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:17:23] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:17:32] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:17:35] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:17:36] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:18:11] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:18:12] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:18:19] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [05/Aug/2020 23:18:21] "[37mGET / HTTP/1.1[0m" 200 -
103.208.178.194 - - [06/Aug/2020 00:10:54] "[33mGET /admin/login.asp HTTP/1.1[0m" 404 -
185.39.11.105 - - [06/Aug/2020 00:54:05] "[33mPOST /boaform/admin/formLogin HTTP/1.1[0m" 404 -
164.70.183.224 - - [06/Aug/2020 01:15:45] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:15:46] "[37mGET /signup/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:15:53] "[31m[1mPOST /signup/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [06/Aug/2020 01:21:03] "[31m[1mPOST /signup/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [06/Aug/2020 01:21:06] "[31m[1mPOST /signup/ HTTP/1.1[0m" 401 -
164.70.183.224 - - [06/Aug/2020 01:21:09] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:21:13] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:21] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:23] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:25] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:27] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:29] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:30] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:30] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:49] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:52] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:35:58] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:36:16] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:36:32] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:36:42] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:36:44] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:37:32] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:37:37] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:23] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:30] "[35m[1mGET /restaurant/1 HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib64/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib64/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dbuser/dblec_app/api/views/user.py", line 272, in showRestaurantReports
    user_id = current_user.id
  File "/usr/local/lib/python3.6/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'Anonymous' object has no attribute 'id'
164.70.183.224 - - [06/Aug/2020 01:38:30] "[37mGET /restaurant/1?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:30] "[37mGET /restaurant/1?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:30] "[37mGET /restaurant/1?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:31] "[37mGET /restaurant/1?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:31] "[37mGET /restaurant/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:31] "[37mGET /restaurant/1?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:55] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:38:59] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:01] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:03] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:04] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:06] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:09] "[37mGET /user/matsuoka/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:46] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:39:50] "[37mGET /logout/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:40:11] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:40:32] "[37mGET /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:41:20] "[37mPOST /login/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:41:34] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:41:45] "[37mGET /user/matsuoka/following HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:41:51] "[37mGET /user/matsuoka/followed HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:42:03] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:42:14] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:42:18] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:10] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:15] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:19] "[37mGET /user/matsuoka HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:24] "[37mGET /user/dbuser HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:49] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:50] "[37mGET /user/mizutani HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:44:57] "[37mGET /user/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:04] "[37mGET /restaurant/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:12] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:15] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:20] "[37mGET /restaurant/1 HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:25] "[37mGET /restaurant/3 HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:28] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:45:46] "[37mGET / HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:46:25] "[37mPOST /report/ HTTP/1.1[0m" 200 -
164.70.183.224 - - [06/Aug/2020 01:46:28] "[37mGET /restaurant/2 HTTP/1.1[0m" 200 -
46.98.130.16 - - [06/Aug/2020 03:13:50] "[37mGET / HTTP/1.1[0m" 200 -
89.248.174.166 - - [06/Aug/2020 03:15:35] "[37mGET / HTTP/1.1[0m" 200 -
185.39.11.105 - - [06/Aug/2020 04:11:34] "[37mGET / HTTP/1.1[0m" 200 -
185.39.11.105 - - [06/Aug/2020 04:41:13] "[37mGET / HTTP/1.1[0m" 200 -
80.82.78.85 - - [06/Aug/2020 04:57:17] "[37mGET / HTTP/1.1[0m" 200 -
178.44.190.199 - - [06/Aug/2020 05:10:16] "[37mGET / HTTP/1.1[0m" 200 -
163.172.66.130 - - [06/Aug/2020 06:49:06] "[37mGET / HTTP/1.1[0m" 200 -
193.118.53.194 - - [06/Aug/2020 06:56:02] "[37mGET / HTTP/1.1[0m" 200 -
188.215.218.93 - - [06/Aug/2020 07:58:09] "[37mGET / HTTP/1.1[0m" 200 -
105.154.28.195 - - [06/Aug/2020 08:06:32] "[37mGET / HTTP/1.1[0m" 200 -
105.154.28.195 - - [06/Aug/2020 08:06:32] "[37mGET / HTTP/1.1[0m" 200 -
105.154.28.195 - - [06/Aug/2020 08:06:32] "[37mGET / HTTP/1.1[0m" 200 -
93.175.201.36 - - [06/Aug/2020 08:43:32] "[37mGET / HTTP/1.1[0m" 200 -
